src/constants.js

```diff
@@ -3,4 +3,6 @@
 export default {
     // The CyVerse page for DE. Not the DE base URL.
     CYVERSE_URL: "https://cyverse.org/discovery-environment",
+    COMMUNITY_DATA_PATH: "/iplant/home/shared",
+    SHARED_WITH_ME_PATH: "/iplant/home",
```

Comment:
We can save this for another PR, but these values might be better suited coming from the external config file

Reply:
Yes. I have to file some issues against data-info. One of this is modifying response to the `roots` endpoint. I think lot hard coded checks in this PR can be fixed with some changes to `roots` endpoint

Reply:
These paths are already coming from the roots endpoint now?

In the `Listing` API call, these roots are detected based on their labels, but they're only being put into the `dataRoots` list. Could these paths also be cached the same as `userHomePath` and `userTrashPath`?

Reply:
I think I  understand what you saying but let me try it out and see :P